### PR TITLE
Block Editor Views Annotations

### PR DESCRIPTION
**Pull request by bordoni**
_Ref:_ [C#118663](http://central.tri.be/issues/118663)
---

### CODE REVIEW

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$label' | [back-link.php:23](https://github.com/moderntribe/the-events-calendar/blob/c5df1804e4d82f4d60aa58e6be082b542333c491/src/views/editor/parts/back-link.php#L23)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$events_label_plural' | [back-link.php:23](https://github.com/moderntribe/the-events-calendar/blob/c5df1804e4d82f4d60aa58e6be082b542333c491/src/views/editor/parts/back-link.php#L23)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$events_label_singular' | [footer.php:21](https://github.com/moderntribe/the-events-calendar/blob/c5df1804e4d82f4d60aa58e6be082b542333c491/src/views/editor/parts/footer.php#L21)

> via jenkins/codesniffer-phpcs

File: src/views/single-event/back-link.php [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<?php
```
Comment (mitogh) [inline src/views/single-event/back-link.php]: Are this file being used?

Comment (mitogh) [reply src/views/single-event/back-link.php]: Same question for the rest of files after this one.

Comment (bordoni) [reply src/views/single-event/back-link.php]: Yeah loading from `single-event-blocks`

**Pull request merged**